projects/2FAI/index.js

```diff
@@ -0,0 +1,26 @@
+const { stakings } = require("../helper/staking");
+
+const revShareContract = [
+    "0x394a95251135274165837be000d771D5DCb08e15",
+];
+
+const tokenVestingUNCX = [
+    "0xA82685520C463A752d5319E6616E4e5fd0215e33"
+]
+
+const twoFAI = "0x1C1eC1bb5f12F24c97231165B13F3Eab9d4Ec00e"
+
+const treasury = [
+    "0x5012f99fe002EA044492a59826026C88A96d535f" 
+]
+
+module.exports = {
+  misrepresentedTokens: true,
+  base: {
+    treasury: stakings(treasury, twoFAI, 'base'),
+    incentivized: stakings(revShareContract, twoFAI, 'base'),
```

Comment:
hmm, this can be counted as staking, but these tokens are not staked by the users? I dont see any method where user can deposit tokens here

Reply:
These tokens are deposited from treasury and claimed by users that are elegible for rev share

Reply:
yes, but they are not staked by users.